subsys/bluetooth/controller/Kconfig

```diff
@@ -200,6 +207,13 @@ config BLUETOOTH_CONTROLLER_SCHED_ADVANCED
 	  Disabling this feature will lead to overlapping role in timespace
 	  leading to skipped events amongst active roles.
 
+config BLUETOOTH_CONTROLLER_TIFS_HW
```

Comment:
Unless you have a very good reason not to use the hw TIFS switching I don't see why you couldn't remove this Kconfig option and use the hardware when available (compiling for nRF52 and higher)